components/places/ios/Places/Places.swift

```diff
@@ -339,17 +339,17 @@ public class PlacesReadConnection {
      * - Parameter keyword: The search keyword.
      * - Returns: The bookmarked URL for the keyword, if set.
      * - Throws:
-     *     - `PlacesError.databaseInterrupted`: If a call is made to `interrupt()` on this
-     *                                          object from another thread.
-     *     - `PlacesError.connUseAfterAPIClosed`: If the PlacesAPI that returned this connection
-     *                                            object has been closed. This indicates API
-     *                                            misuse.
-     *     - `PlacesError.databaseBusy`: If this query times out with a SQLITE_BUSY error.
-     *     - `PlacesError.unexpected`: When an error that has not specifically been exposed
-     *                                 to Swift is encountered (for example IO errors from
-     *                                 the database code, etc).
-     *     - `PlacesError.panic`: If the rust code panics while completing this
-     *                            operation. (If this occurs, please let us know).
+     *     - `PlacesApiError.databaseInterrupted`: If a call is made to `interrupt()` on this
+     *                                             object from another thread.
+     *     - `PlacesConnectionError.connUseAfterAPIClosed`: If the PlacesAPI that returned this connection
+     *                                                      object has been closed. This indicates API
+     *                                                      misuse.
```

Comment:
I'm not sure how these doc strings should be formatted.  Now that the errors use a couple extra chars, if you align everything after the colon, there's very little space to use.